site/docs/v1/tech/admin-guide/upgrade.adoc

```diff
@@ -4,44 +4,300 @@ title: Upgrade FusionAuth
+
+
+[WARNING]
+====
+Note that Maintenance Mode should never be used in production.
+
+You should always back up your database before upgrading regardless of the upgrade method you choose.
 ====
 
 
-== ZIP Packages
+In production, you may want to perform the database upgrade manually, also known as an out-of-band upgrade. This is useful if you want to perform the database upgrade in a way that minimizes downtime or as part of a larger automated or coordinated deployment. Upgrading the database manually involves running SQL scripts directly on your database to update the database schema. 
+
+Depending on your current version and the new version you will be updating to, you might need to execute one or more SQL scripts to update your database. These scripts are located in the migrations folder in the Database Schema ZIP file, which you can download from the https://fusionauth.io/direct-download[Direct Downloads] page.
+
+[WARNING]
+====
+When upgrading your database from a previous version, be sure to only run the scripts located in the `migrations` folder. The base files
+ `mysql.sql` and `postgresql.sql` should only be used during a clean installation when no database schema is present.
+====
+
+Find the FusionAuth migrations in the Database Schema ZIP file. Run them in order, starting with the first migration greater than the version you are coming from, and ending with the version that is less than or equal to the target upgrade version.
+
+For example, if upgrading from version `1.19.1` to `1.22.0`, run the following SQL migrations in this order:
+
+* `1.20.0.sql`
+* `1.21.0.sql`
+* `1.22.0.sql`
+
+[source]
+----
+fusionauth-database-schema/
+|-- migrations/
+    |--  [mysql | postgresql]/
+      |-- 1.1.0.sql
+      |-- 1.2.0.sql
+      |-- 1.3.0.sql
+      |-- 1.3.1.sql
+      |-- 1.5.0.sql
+      |-- 1.6.0.sql
+      |-- 1.7.0.sql
+      |-- 1.7.1.sql
+      |-- 1.8.0-RC.1.sql
+      |-- 1.8.1-RC.1.sql
+      |-- 1.11.0.sql
+      |-- 1.12.0.sql
+      |-- 1.13.0.sql
+      |-- 1.14.0.sql
+      |-- 1.15.0.sql
+      |-- 1.15.3.sql
+      |-- 1.16.0-RC.1.sql
+      |-- 1.16.0.sql
+      |-- 1.17.0.sql
+      |-- 1.17.3.sql
+      |-- 1.18.0.sql
+      |-- 1.18.2.sql
+      |-- 1.19.0.sql
+      |-- 1.20.0.sql
+      |-- 1.21.0.sql
+      |-- 1.22.0.sql
+      |-- 1.23.0.sql
+      |-- 1.25.0.sql
+      |-- 1.26.0.sql
+      |-- 1.27.0.sql
+      |-- 1.28.0.sql
+      |-- 1.28.1.sql
+      |-- 1.29.1.sql
+      |-- 1.30.0.sql
+      |-- 1.30.1.sql
+      |-- 1.30.2.sql
+      |-- 1.32.0.sql
+      |-- 1.33.0.sql
+      |-- 1.35.0.sql
+      |-- 1.36.0.sql
+      |-- 1.37.0.sql
+      |-- 1.40.1.sql
+      |-- 1.41.0.sql
+      |-- 1.43.0.sql
+      |-- 1.44.0.sql
+----
+
+
+== FAQs
+
+Q: What is runtime mode and how does it affect upgrades? +  
```

Comment:
Ah, here's the modes stuff I wanted up at the beginning, when I mention that silent mode is brought up in a confusing way.